### PR TITLE
add link to gh pages live preview

### LINKED ISSUE #4: Deploy examples to gh-pages
[Demo-viewer examples](https://github.com/voorhoede/demo-viewer/tree/master/examples) should be deployed to Github pages. We can use Travis CI for this: [deploy to gh-pages script](https://github.com/jbmoelker/nunjucks-bootstrap/blob/master/scripts/deploy.sh). This script also ensures only the master branch is deployed.

Needs an `index.html` with a list of all examples.

### LINKED ISSUE #9: List demo examples in project readme
Link to demo examples (deployed to gh-pages in #4) like done in [Nunjucks Bootstrap > Readme](https://github.com/jbmoelker/nunjucks-bootstrap#modules).

### PR DESCRIPTION
**Pull request by jpsc**
closes #4
closes #9

### CODE REVIEW

**Comment by jbmoelker** [comment]
I'd much prefer examples to live in `dist/examples/` (and still sync `dist/` to gh-pages) (see https://github.com/voorhoede/demo-viewer/commit/99690b0c1532222fbb5fc4c7049f2ab96a54fa0e#commitcomment-18295783).
I'd like to keep a structure where we can have more than one set of demos. So I hope the structure will resemble:

``` markdown
## Live preview

* [Boostrap v3](https://voorhoede.github.io/examples/bootstrap-v3/): <insert meaningful description here>
* <Another example>
```

I'd like to reserve `https://voorhoede.github.io/` for an actual overview page. (something like https://jbmoelker.github.io/serviceworker-introduction/, but with a bit more Voorhoede love).

**Comment by jpsc** [comment]
updated with feedback.
- serving from dist all examples (locally)
- `bootstrap-v3/` will go to bootstrap examples
- fixed link to live example

File: README.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 Preview components in isolation with documentation, annotations and breakpoint controls.
 
+## Live preview
+
+* [Bootstrap 3](https://voorhoede.github.io/bootstrap-v3)
```
Comment (jbmoelker) [inline README.md]: This URL won't work. Guessing it should be `https://voorhoede.github.io/demo-viewer/bootstrap-v3/`. Better yet, I'd like it to be `https://voorhoede.github.io/demo-viewer/examples/bootstrap-v3/` (will elaborate in general comment).

**Comment by jbmoelker** [comment]
@jpsc nice! small things:
- to me "copy" is an implementation of "build" (could later be replaced), so I'd rename `"copy:examples"` to `"build:examples"`.
- there is no specific reason why we can't generate the index page at the same time we generate the examples. So I'd rename that to `"build:index", which you would then also execute from`"build"`.
- I'd prefer examples to end up in `dist/examples/` instead of `dist/` so that we will never run into directory / naming conflicts and structure will resemble `src/`.

**Pull request merged**